src/models/department.ts

```diff
@@ -0,0 +1,19 @@
+import Position from "./position";
+import Employee from "./employee";
+import { Prop, PropType, Ref } from "@typegoose/typegoose";
+
+class Department {
+    @Prop({ required: true, type: () => String }) //? Should maybe be unique?
```

Comment:
We can make it unique, but also I don't think it's a huge deal - they probably won't be adding departments very often and they probably won't accidentally create two departments with the same name